packages/typespec-autorest-canonical/src/lib.ts

```diff
@@ -0,0 +1,273 @@
+import { createTypeSpecLibrary, JSONSchemaType, paramMessage } from "@typespec/compiler";
+
+export interface AutorestCanonicalEmitterOptions {
+  /**
+   * Name of the output file.
```

Comment:
Make sure this doc match the string in the json schema below

Reply:
updated